setup.py

```diff
@@ -61,7 +61,7 @@ def read_md(f):
         'bin/wiotp-cli'
     ],
     url='https://github.com/ibm-watson-iot/iot-python',
-    license=open('LICENSE').read(),
+    license='Eclipse Public License 1.0 (EPL-1.0)',
```

Comment:
```suggestion
    license='EPL-1.0',
```
Might be even better to just use the SPDX license identifier directly.
https://spdx.org/licenses/EPL-1.0.html

Reply:
For sure, didn't know about this yet 3 years ago

Reply:
took a bit longer, you cannot accept suggestions on a phone 😅